sbom/examples/rpm/build/from-koji.py

```diff
@@ -165,7 +170,7 @@ def mock_openssl_midstream(sfn, source, sname, sver):
     )
 
     # Construct the URL for the sourceN package
-    url = f"https://github.com/(RH openssl midstream repo)/archive/refs/tags/{sver}.{ext}"
+    url = f"https://github.com/(RH {sname} midstream repo)/archive/refs/tags/{sver}.{ext}"
```

Comment:
If `ext` is an empty string here, it produces a URL with a trailing dot.

Reply:
I fixed this in the latest version.